content/en/spec/activitypub.md

```diff
@@ -536,19 +536,19 @@ Public keys are used for HTTP Signatures and Linked Data Signatures. This is imp
     "https://www.w3.org/ns/activitystreams",
     "https://w3id.org/security/v1"
   ],
-  "id": "https://mastodon.social/users/Gargron",
+  "id": "https://example.com/users/Gargron",
```

Comment:
same comment re: mastodon behavior but also this was specifically an example of real data generated by mastodon